vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -124,7 +145,11 @@ def _action_inputs(self) -> dict[str, _IdProperty]:
 
     def __call__(self, options):
         full_options, default_value = get_options_and_default(options=options, multi=self.multi)
-        option_height = _calculate_option_height(full_options)
+        # 30 characters is roughly the number of "A" characters you can fit comfortably on a line in the page dropdown
+        # (placed on the left-side). 15 is half this width for when the dropdown is in a container's controls.
+        # "A" is representative of a slightly wider than average character:
+        # https://stackoverflow.com/questions/3949422/which-letter-of-the-english-alphabet-takes-up-most-pixels
+        option_height = _calculate_option_height(full_options, 15 if self._in_container else 30)
```

Comment:
Note this is now recalculated with 15 or 30 every time the page is refreshed and `__call__` is executed. Previously it would only be set to 15 upfront and then switch back to 30 after that.